libs/zopengl/src/wrapper.zig

```diff
@@ -0,0 +1,2624 @@
+pub const ARRAY_BUFFER = bindings.ARRAY_BUFFER;
+pub const ELEMENT_ARRAY_BUFFER = bindings.ELEMENT_ARRAY_BUFFER;
+pub const ARRAY_BUFFER_BINDING = bindings.ARRAY_BUFFER_BINDING;
+pub const ELEMENT_ARRAY_BUFFER_BINDING = bindings.ELEMENT_ARRAY_BUFFER_BINDING;
+pub const VERTEX_ATTRIB_ARRAY_BUFFER_BINDING = bindings.VERTEX_ATTRIB_ARRAY_BUFFER_BINDING;
+pub const READ_ONLY = bindings.READ_ONLY;
+pub const WRITE_ONLY = bindings.WRITE_ONLY;
+pub const READ_WRITE = bindings.READ_WRITE;
+pub const BUFFER_ACCESS = bindings.BUFFER_ACCESS;
+pub const BUFFER_MAPPED = bindings.BUFFER_MAPPED;
+pub const BUFFER_MAP_POINTER = bindings.BUFFER_MAP_POINTER;
+pub const STREAM_DRAW = bindings.STREAM_DRAW;
+pub const STREAM_READ = bindings.STREAM_READ;
+pub const STREAM_COPY = bindings.STREAM_COPY;
+pub const STATIC_DRAW = bindings.STATIC_DRAW;
+pub const STATIC_READ = bindings.STATIC_READ;
+pub const STATIC_COPY = bindings.STATIC_COPY;
+pub const DYNAMIC_DRAW = bindings.DYNAMIC_DRAW;
+pub const DYNAMIC_READ = bindings.DYNAMIC_READ;
+pub const DYNAMIC_COPY = bindings.DYNAMIC_COPY;
+pub const SAMPLES_PASSED = bindings.SAMPLES_PASSED;
+pub const SRC1_ALPHA = bindings.SRC1_ALPHA;
+
+// pub var genQueries: *const fn (n: Sizei, ids: [*c]Uint) callconv(.C) void = undefined;
+// pub var deleteQueries: *const fn (n: Sizei, ids: [*c]const Uint) callconv(.C) void = undefined;
+// pub var isQuery: *const fn (id: Uint) callconv(.C) Boolean = undefined;
+// pub var beginQuery: *const fn (target: Enum, id: Uint) callconv(.C) void = undefined;
+// pub var endQuery: *const fn (target: Enum) callconv(.C) void = undefined;
+// pub var getQueryiv: *const fn (target: Enum, pname: Enum, params: [*c]Int) callconv(.C) void = undefined;
+// pub var getQueryObjectiv: *const fn (id: Uint, pname: Enum, params: [*c]Int) callconv(.C) void = undefined;
+// pub var getQueryObjectuiv: *const fn (id: Uint, pname: Enum, params: [*c]Uint) callconv(.C) void = undefined;
+
+// pub var bindBuffer: *const fn (target: Enum, buffer: Uint) callconv(.C) void = undefined;
+pub fn bindBuffer(target: BufferTarget, buffer: Buffer) void {
+    assert(@bitCast(Uint, buffer) > 0);
+    bindings.bindBuffer(@enumToInt(target), @bitCast(Uint, buffer));
+}
+
+// pub var deleteBuffers: *const fn (n: Sizei, buffers: [*c]const Uint) callconv(.C) void = undefined;
+
+// pub var genBuffers: *const fn (n: Sizei, buffers: [*c]Uint) callconv(.C) void = undefined;
+pub fn genBuffer(ptr: *Buffer) void {
+    bindings.genBuffers(1, @ptrCast([*c]Uint, ptr));
+}
+pub fn genBuffers(buffers: []Buffer) void {
+    bindings.genBuffers(buffers.len, @ptrCast([*c]Uint, buffers.ptr));
+}
+
+// pub var isBuffer: *const fn (buffer: Uint) callconv(.C) Boolean = undefined;
+
+// pub var bufferData: *const fn (
+//     target: Enum,
+//     size: Sizeiptr,
+//     data: ?*const anyopaque,
+//     usage: Enum,
+// ) callconv(.C) void = undefined;
+pub fn bufferData(
+    target: BufferTarget,
+    size: usize,
+    maybe_bytes: ?[]const u8,
```

Comment:
Maybe? This would be consistent with `texImage2D()`?
```suggestion
    maybe_bytes: ?[*]const u8,
```